projects/core/src/error-handling/error-interceptors/error-interceptor.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { InjectionToken } from '@angular/core';
+
+/**
+ * ChainedErrorInterceptorFn is a function that is passed to the error interceptor.
+ * It is responsible for passing the error to the next interceptor in the chain if provided.
```

Comment:
[MINOR]
What about: 
```suggestion
 * It is responsible for handling an error in some way. If the error is not resolved by this interceptor, it should pass the original error (or a new error) to the next interceptor in the chain.
```